emails/models.py

```diff
@@ -686,6 +686,14 @@ class DomainAddrNeedSubdomainException(CannotMakeAddressException):
     status_code = 400
 
 
+class DomainAddrUpdateException(CannotMakeAddressException):
+    """Exception raised when attempting to edit an existing domain address field."""
+
+    default_code = "address_exists"
```

Comment:
```suggestion
    default_code = "address_not_editable"
```
The previous error code makes it confusing to another error where the domain address cannot be created because an address exists.

Reply:
Addressed these changes, https://github.com/mozilla/fx-private-relay/pull/4519/commits/08417ecce9da7ceb8eb1d70928ce07ed8d5c2210